nebula/ui/components/VPNMouseArea.qml

```diff
@@ -11,7 +11,8 @@ MouseArea {
 
     property var targetEl: parent
     property var uiState: VPNTheme.theme.uiState
-    property var onMouseAreaClicked: function() { parent.clicked() }
+    property bool propagateClickToParent: true
```

Comment:
`false` by default?

Reply:
It probably should be `false` by default, but the current functionality always propagates the click to the parent, so I left that as the default functionality to avoid having to make changes everywhere this component is used - mostly to avoid churn while I still feel my way around the codebase. 

I could change this though if you think it's worth it?

Reply:
All good to have to `false`